packages/clarity/test/nativeClarityBinProvider.ts

```diff
@@ -0,0 +1,47 @@
+import { assert } from "chai";
+import fs from "fs-extra";
+import { NativeClarityBinProvider, Receipt } from "../src";
+import { getTempFilePath } from "../src/utils/fsUtil";
+
+describe("NativeClarityBinProvider", () => {
+  it("create ephemeral", async () => {
+    const provider = await NativeClarityBinProvider.createEphemeral(
```

Comment:
`NativeClarityBinProvider.createEphemeral(getDefaultBinaryFilePath())` should work once this is rebased off latest master